### PR TITLE
Makeshift revolvers and pistols 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
##  About The Pull Request

Adds makeshift revolver that can be calibrated like detectives but at grater risks and one way!
Adds makeshift pistols that also can be calibrated one way, at grater risks

## Why It's Good For The Game

More ways to revs to VIVA harder without needing cargo on lock and key, more guns that are frankly shit and should not be used outside of 1-2 cases when needed. - `Just make a shotgun`  
But its holdout kinda guns so it can be usefull I guess
Flat guns can no longer be suppressed do to missing sprites/Idk if it was even meant to
## Changelog
:cl:
add: Makeshift Revolvers and pistols
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/pistol.dm [hunk 1]
```diff
@@ -80,6 +80,7 @@
 	name = "flat gun"
 	desc = "A 2 dimensional gun.. what?"
 	icon_state = "flatgun"
+	can_suppress = FALSE
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/ballistic/pistol.dm:83]: Flat guns down have sprites by the by for suppress, also the stick man fire .45 but I dont want to change that here

**Comment by ghost** [comment]
revolver is kinda sketchy but i support the pistol

**Comment by DeltaFire15** [comment]
Yeah uh could we like not give people a 2hit ko revolver that they can easily make? Thanks.

**Comment by Trilbyspaceclone** [comment]
>revolver looks bad
*stares at slugs that dont fail*

**Comment by DeltaFire15** [comment]
Improv shotguns can't store 7 slugs in them but need time to reload after each shot.

**Comment by Trilbyspaceclone** [comment]
the revolver is also 1 shot at a time that is single loaded and needs to be unloaded like that too

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 2]
```diff
@@ -364,3 +364,127 @@
...
 		user.Knockdown(80)
+
+// IMPROVISED REVOLVER //
+//Has 3 calibers - .38 .357 rev762.
+//.38 has a 5% odds of failing, .357 has a 25% as well as rev762
+//Once you caliber up, you can NOT go down
+
+/obj/item/gun/ballistic/revolver/detective/makeshift
+	name = "makeshift revolver"
+	desc = "A makeshift revolver made with ducttape and dreams. Not the most reliable..."
```
Comment (Arturlang) [inline code/modules/projectiles/guns/ballistic/revolver.dm]: there needs to be a space between duct and tape

**Comment by silicons** [comment]
this is kinda silly and not in the good way

**Comment by Trilbyspaceclone** [comment]
@kevinz000 How so?
Its just a way for people to make more weapons. Its not as good as a shotgun and has fail odds, that really mess over the user. If you want a sure fire way to kill someone you use the shotgun that you can make with less with slugs and stuff

**Comment by Tupinambis** [comment]
@Trilbyspaceclone Do this, but make it a pipe rifle instead. By pipe rifle I basically mean a craftable mosin. It would be much more balanced while still being fairly useful.

**Comment by Ghommie** [comment]
As long as gunsmithing is erroneously narrowed down to sticking various pieces together with wrapping paper and not given a proper feature, I won't trust these sort of changes won't have a negative impact to the game.

Also the screwdriver action to change caliber for the makeshift revolver is totally nuts and defies sense.

**Pull request closed without merging**